test/modules/layers/test_attention.py

```diff
@@ -125,22 +129,56 @@ def test_combine_multihead(self):
 
     def test_multi_head_attention(self):
         # New tensors because need unflattened shape
-        q = torch.randn(1, *self.input_shape, self.hidden_dim)
-        k = torch.randn(1, *self.input_shape, self.hidden_dim)
-        v = torch.randn(1, *self.input_shape, self.hidden_dim)
+        q = 2 * torch.ones(1, *self.input_shape, self.hidden_dim)
+        k = 2 * torch.ones(1, *self.input_shape, self.hidden_dim)
+        v = 2 * torch.ones(1, *self.input_shape, self.hidden_dim)
         actual = self.mha(q, k, v)
         expected = torch.tensor(
             [
                 [
                     [
-                        [[-0.1824, 0.2826, 0.4706], [-0.1540, 0.2962, 0.4301]],
-                        [[-0.1795, 0.2889, 0.4178], [-1.2837, -0.2228, -0.6794]],
+                        [[2.4187, 4.1634, 1.4579], [2.4187, 4.1634, 1.4579]],
+                        [[2.4187, 4.1634, 1.4579], [2.4187, 4.1634, 1.4579]],
                     ],
                     [
-                        [[-0.5227, 0.1744, 0.3691], [-0.3784, 0.2148, 0.3581]],
-                        [[-1.0747, -0.1513, -0.4717], [-1.3936, -0.2522, -0.7915]],
+                        [[2.4187, 4.1634, 1.4579], [2.4187, 4.1634, 1.4579]],
+                        [[2.4187, 4.1634, 1.4579], [2.4187, 4.1634, 1.4579]],
                     ],
                 ]
             ]
         )
         assert_expected(actual, expected, rtol=0, atol=1e-4)
+
+    def test_axial_block_mha_length(self):
+        """Test AxialAttentionBlock number of MHAs"""
+        assert len(self.block.mha_attns) == 3, "incorrect number of MHAs"
```

Comment:
Do we really need this test? Feels like it is just testing that the test initialization is working as expected